config/jobs/kubernetes/sig-release/cip/container-image-promoter.yaml

```diff
@@ -21,6 +21,27 @@ presubmits:
         # /home/prow/go/src/github.com/{{.Org}}/{{.Repo}}, so just '.' should
         # suffice, but it's nice to be explicit.
         - -thin-manifest-dir=/home/prow/go/src/github.com/kubernetes/k8s.io/k8s.gcr.io
+  # Check that images to be promoted are free of vulnerabilities
+  - name: pull-k8sio-cip-vuln
+    annotations:
+      testgrid-create-test-group: 'true'
+      testgrid-dashboards: wg-k8s-infra-k8sio
+    decorate: true
+    skip_report: false
+    run_if_changed: 'k8s.gcr.io/((images/.*/images\.yaml)|(manifests/.*/promoter-manifest\.yaml))'
+    max_concurrency: 10
+    branches:
+    - ^vuln-check-test$
+    spec:
+      serviceAccountName: k8s-infra-gcr-vuln-scanning
```

Comment:
https://github.com/kubernetes/test-infra/blob/master/config/prow/cluster/build_serviceaccounts.yaml#L21-L28 needs to be updated to use this shorter name

Reply:
Done